### ISSUE TITLE
improvement to get json playerConfig

### ISSUE THREAD

**Issue by utilsites** [datarhei/restreamer-ui#18]
Restreamer has a url to get player configs:
_https://[restreamer-domain]/channels/[GUID]/config.js_

this returns the following script file (example):
`var playerConfig = {"autoplay":false,"mute":true, ... }}`

I think it would be better to **just return the json part**, without "var playerConfig = ", that way it would be much more interoperable, without needing to instantiate script to get a var name, that is more hardcoded.

**Comment by ioppermann** [comment]
This is not meant to be an API to get the config for the player from. It's a file as part of the page where the player is loaded.

**Comment by utilsites** [comment]
Right, but if I want to use a different video webplayer than videojs or add some player plugins without changing Restreamer code, this would be very useful.
Thanks for your comment.

**Comment by ShHaWkK** [comment]
I kept the existing `config.js` behavior intact and opened #109 with a backward-compatible `config.json` alongside it. The full test suite and production build pass on the branch.
